titus-api/src/main/java/com/netflix/titus/api/jobmanager/TaskAttributes.java

```diff
@@ -58,6 +58,7 @@
     public static final String TASK_ATTRIBUTES_CONTAINER_IP = "task.containerIp";
     public static final String TASK_ATTRIBUTES_CONTAINER_IPV4 = "task.containerIPv4";
     public static final String TASK_ATTRIBUTES_CONTAINER_IPV6 = "task.containerIPv6";
+    public static final String TASK_ATTRIBUTES_TRANSITION_IPV4 = "task.transitionIPv4";
```

Comment:
Can you add a javadoc for what this is? The others are self explanatory but this one could be interpreted in different ways.